app/src/main/java/org/eyeseetea/malariacare/domain/entity/Credentials.java

```diff
@@ -56,4 +58,12 @@ public int hashCode() {
                 password.hashCode();
     }
 
+    public boolean isLogged() {
```

Comment:
You can ignore it at this moment, this is solved in the next PR ( https://github.com/EyeSeeTea/pictureapp/pull/980 ) This is only a temporal solution.